libs/sdk-core/src/lib/rest/rest.ts

```diff
@@ -148,4 +148,41 @@ export class Rest implements IRest {
       }),
     );
   }
+
+  getStream(path: string, body: any): Observable<Uint8Array> {
+    path = this.getFullUrl(path);
+    return new Observable<Uint8Array>((observer) => {
+      fetch(path, { method: 'POST', headers: this.getHeaders('POST', path), body: JSON.stringify(body) }).then(
+        (res) => {
+          const reader = res.body?.getReader();
+          if (!reader) {
+            observer.next(new Uint8Array());
+            observer.complete();
+          } else {
+            let data = new Uint8Array();
+            const readMore = () => {
```

Comment:
why did you create this function? why not directly running the code it contains?

Reply:
because this function calls itself recursively

Reply:
right, I missed one call